trace_decoder/src/compact/compact_prestate_processing.rs

```diff
@@ -1221,6 +1262,7 @@ const fn try_get_node_entry_from_witness_entry(entry: &WitnessEntry) -> Option<&
     }
 }
 
+#[allow(dead_code)]
 #[derive(Debug)]
 enum TraverserDirection {
```

Comment:
Do we plan on using it? Otherwise I'd just go for removing it

Reply:
Yeah, also remove. We ended up only using `Forward` IIRC.